### PR TITLE
Pipeline, Liubov Gorbunova - 21FPL2

### PR DESCRIPTION
**Pull request by lubagorbunova**

### CODE REVIEW

File: lab_6_pipeline/pipeline.py [hunk 1]
```diff
@@ -18,21 +36,59 @@ def __init__(self, path_to_raw_txt_data: Path):
...
+        #for file in meta_files:
+        #    if not file.stat().st_size:
+        #        raise InconsistentDatasetError
+        for file in raw_files:
+            if not file.stat().st_size:
+                raise InconsistentDatasetError
+        for i in range(max_number):
+            filename_raw = path + '\\' + str(i+1) + '_raw.txt'
+        #    filename_meta = path + '\\' + str(i + 1) + '_meta.json'
+            if filename_raw not in raw_files_list: #or filename_meta not in meta_files_list:
```
Comment (dmitry-uraev) [inline lab_6_pipeline/pipeline.py]: you may omit comments with not used code

Comment (dmitry-uraev) [reply lab_6_pipeline/pipeline.py]: just to shorten implementation

File: main.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import os.path
```
Comment (dmitry-uraev) [inline main.py]: please, remove files that are not related to labs implementation

**Review by dmitry-uraev — COMMENTED** [review]
Good! You may try to improve target_score!

File: main.py (hunk 2 above)
Comment (lubagorbunova) [reply main.py]: done

**Review by dmitry-uraev — APPROVED** [review]
Good! Done with score 4

File: lab_6_pipeline/pipeline.py (hunk 1 above)
Comment (lubagorbunova) [reply lab_6_pipeline/pipeline.py]: I want to use this code later, as there are more checks in this method for higher marks
